warehouse/query-core/src/main/java/datawave/query/jexl/visitors/LiteralNodeVisitor.java

```diff
@@ -35,4 +38,47 @@ public Object visit(ASTEQNode node, Object data) {
         return literals;
     }
     
+    // Descend through these nodes
+    @Override
+    public Object visit(ASTJexlScript node, Object data) {
+        node.childrenAccept(this, data);
+        return data;
+    }
+    
+    @Override
+    public Object visit(ASTAndNode node, Object data) {
```

Comment:
same comment as above.

Reply:
For some reason, I thought BaseVisitor blocked but must have been an oversight.